### DISCUSSION TITLE
Taskbar gets permanently stuck visually after running fullscreen apps/games.

### DISCUSSION THREAD

**Discussion by notNSANE** [valinet/ExplorerPatcher#3224] [Q&A]
I need to restart explorer.exe every single time this issue happens.

It might be related to the disappearing icons bug, which is still happening but less often. I use 2 monitors with 2 taskbars enabled.

Windows 11 x64, Explorer patcher in Windows 11 mode.

**Comment by pyrates999** [comment]
Can you show a screen shot of what you mean?

**Comment by notNSANE** [comment]
there's no easy way to show it, but just imagine your current taskbar, then
you switch the virtual desktop and you get the same taskbar than the other
desktop.

**Comment by pyrates999** [comment]
Lets get some general info then.

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by notNSANE** [comment]
Sorry for the long delay, I reinstalled the AMD driver and used windows 11 taskbar to make sure it wasn't that - it didn't help.

Winver: 22631.3527
Explorer patcher: 22621.3527.6.2

![image](https://github.com/valinet/ExplorerPatcher/assets/7162375/0b56b911-57cc-4cad-9bf1-9ac437a2da95)

![image](https://github.com/valinet/ExplorerPatcher/assets/7162375/0fb26127-d528-42af-8bc0-53a18c8c30e3)

![image](https://github.com/valinet/ExplorerPatcher/assets/7162375/32e593e4-74da-424f-94a8-f66a6b744ab2)

![image](https://github.com/valinet/ExplorerPatcher/assets/7162375/506815dd-afd8-4a70-bd22-1d980aedfde5)

![image](https://github.com/valinet/ExplorerPatcher/assets/7162375/51aa138b-2d3e-44cf-8755-2e8b0afd4e7f)


Right now I'm using W10 taskbar because the issue got extremely annoying, happening several times a day.

**Comment by pyrates999** [comment]
Does this happen if you uninstall EP?

**Reply by notNSANE** [reply]
yeah, I installed Explorer Patcher try and solve the issue, haha. I haven't seen any issues with W10 taskbar and it's serviceable. I'm getting 2 notifications per email, tho.

**Reply by pyrates999** [reply]
Then this is not something for EP, sorry.  Microsoft has to fix this.
